chassis/core/src/main/java/com/griddynamics/jagger/master/MasterToJaasCoordinator.java

```diff
@@ -31,7 +32,7 @@
  * Handles Jagger Master node to JaaS communication -
  * initial registration and further status updates with commands parsing from JaaS response.
  */
-public class MasterToJaasCoordinator {
+public class MasterToJaasCoordinator implements Closeable {
```

Comment:
when masterToJaasCoordinator will be interrupted it will be not visible in the logs. we will silently exit standBy mode => difficult to analyze logs